src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -746,6 +747,38 @@ public static int getPixelValue(String value) {
 		return result;
 	}
 
+	public static int[] parseBitsPerSample(String value) {
+		if (isBlank(value)) {
+			LOGGER.debug("MediaInfo returned a blank bits per sample value, falling back to default");
+			return new int[0];
```

Comment:
@Sami32 I interpret "not valid" or "invalid" as broader than blank. Blank to me means ```null``` or a string containing nothing or only whitespace. Empty means only ```null``` or an empty string. I guess I'm influenced by Apache Commons, as it's the definition they use. Invalid on the other hand is any value that's not "useful"/"meaningful" in the current content, and includes most non-empty string values as well.